content/en/docs/reference/node/kernel-version-requirements.md

```diff
@@ -0,0 +1,68 @@
+---
+content_type: "reference"
+title: Linux Kernel Version Requirements
+weight: 10
+---
+
+{{% thirdparty-content %}}
+
+Many features rely on specific kernel functionalities and have minimum kernel version requirements.
+
+## Pod sysctls {#requirements-pod-sysctl}
```

Comment:
Would be great to have comment in this page pointing back to the constants in kubelet for each of the mentioned limitations. This way maintaining this page will be easier long term

```
<!--
Code: link to the constant in k/k
-->
```

Reply:
https://github.com/kubernetes/website/pull/47363/commits/19a4a014b74ba5e751239840f32d52be087a4cce Added some code reference.